plugin.yaml

```diff
@@ -207,6 +223,13 @@ relationships:
 #                 - establish: cloudstack.neutron_plugin.router.connect_subnet
 #                 - unlink: cloudstack.neutron_plugin.router.disconnect_subnet
 
+    cloudify.cloudstack.volume_contained_in_vm:
```

Comment:
volume should be attached to vm, not contained in.

have a look at the openstack example:
https://github.com/cloudify-cosmo/cloudify-openstack-plugin/blob/master/plugin.yaml#L297